### PR TITLE
Adding subkey knowledge base

### PR DESCRIPTION
**Pull request by jimmychu0807**
Thank you for making a PR to the knowledge base. Please make sure it follows these short guidelines:

- [ ] Are the audience and objective of the document clear?
- [ ] Is the writing:
  - **Clear**: All jargon defined, explained, or linked to. Best to avoid it.
  - **Precise**: No ambiguous meanings
  - **Concise**: Free of superfluous detail
- [ ] Does it follow our [style guide](/CONTRIBUTING.md#documentation-style)?
- [ ] If this is a new page, does the PR include the appropriate infrastructure, e.g. adding the
  page to a sidebar?
- [ ] Build the page. Does it render properly? E.g. no funny lists or formatting.
- [ ] Links should go to rustdocs or devhub articles, not source code.
- [ ] If this PR addresses an issue in the queue, have you referenced it in the description?

### CODE REVIEW

**Comment by jimmychu0807** [comment]
Updating the `subkey` article with v2.0.0-alpha.5

File: current/integrate/subkey.md [hunk 1]
```diff
@@ -44,212 +44,276 @@ Generate an sr25519 key by running:
...
+flag:
 
 ```bash
-$ subkey --password "correct horse battery staple" generate
-Secret phrase `razor blouse enroll maximum lobster bacon raccoon ocean law question worry length` is account:
-  Secret seed: 0x806e324821f6b09fa0329b7da35b8056b452fdcaadeac463ff2b8db1fc020c3e
-  Public key (hex): 0xe2b410bdc959fcc7f756e6eefdb051000c80be339acd3a8dc6d941211160b266
-  Account ID:       0xe2b410bdc959fcc7f756e6eefdb051000c80be339acd3a8dc6d941211160b266
-  Address (SS58): 5HBxAo81NWBbg6DQNbTdFg7GWQDS1QmkMwNLQso1e11DfQ2Y
+$ subkey --password "pencil laptop kitchen cutter" generate
```
Comment (joepetrowski) [inline current/integrate/subkey.md:128]: I would still like to use `correct horse battery staple` as it's a [well-known](https://xkcd.com/936/) and funny password. Not critical though.

**Review by joepetrowski — APPROVED** [review]
Made a commit for some grammar changes and a second commit to move lines to 100 character limit. Starting to do this on pages as I think they are nicer that way.

Overall looks good though.

File: current/integrate/subkey.md (hunk 1 above)
Comment (jimmychu0807) [reply current/integrate/subkey.md:128]: I am trying to fix this, but then I see there are dependencies below. I used this [secret phrase, seed, password] set to demonstrate other concepts below.

So feel lazy to update them all :)

**Pull request merged**